src/effects/builtin/balanceeffect.cpp

```diff
@@ -77,7 +77,7 @@ BalanceGroupState::BalanceGroupState(const mixxx::EngineParameters& bufferParame
         : EffectState(bufferParameters),
           m_pHighBuf(MAX_BUFFER_LEN),
           m_oldSampleRate(bufferParameters.sampleRate()),
-          m_freq(kMinCornerHz),
+          m_freq(static_cast<int>(kMinCornerHz)),
```

Comment:
m_freq should become a double

Reply:
https://github.com/mixxxdj/mixxx/pull/3227/commits/5b2c009520e1ee3bdd5c29bb477dd09000f51ad0